app/views/internal/classified_listings/index.html.erb

```diff
@@ -5,7 +5,9 @@
   <% if params[:state].present? %>
     <%= hidden_field_tag(:state, params[:state]) %>
   <% end %>
-  <%= submit_tag("Search") %>
+  <%= label_tag(:search, "Filter listings by category") %>
+  <%= select_tag(:filter, options_for_select(["", "cfp", "jobs", "misc", "events", "mentees", "products", "mentors", "education", "collabs", "forhire"], params[:filter])) %>
```

Comment:
I suggest using `ClassifiedListing.categories_available` here to form options. This way we won't have to update in 2 places if the categories change.

Reply:
Thanks @lightalloy! I've updated it to use `ClassifiedListing.categories_available.keys` instead of listing out the options.